code/drasil-example/drasil-example.cabal

```diff
@@ -215,6 +215,7 @@ executable pdController
       Drasil.PDController.Requirements
       Drasil.PDController.SpSysDesc
       Drasil.PDController.TModel
+      Drasil.PDcontroller.Unitals
```

Comment:
This is going to get automatically generated - it is better not to add it by hand (unlike for `package.yaml`)